pipelines/norsk-mirror.yml

```diff
@@ -81,28 +75,22 @@ jobs:
   - get: ci
   - task: check-status-file
     file: ci/tasks/check-norsk-status.yml
+    params:
+      NORSK_PIPELINE: p-concourse-4.2.x
```

Comment:
Probably not... I left the job there just because the norsk pipeline for 4.2.x still exists. But I'll check with Scott if it's really necessary to keep an eye on it or not.

Reply:
I think we don't need to remove it using this PR anyway, I was just wondering while we're at it. maybe we can ask OSL to remove the pipeline as well it is not needed.

Reply:
I'll check with OSL if they can remove it or if they keep for historical reasons.